sdks/python/conftest.py

```diff
@@ -25,19 +24,19 @@
 
 MAX_SUPPORTED_PYTHON_VERSION = (3, 8)
```

Comment:
I wonder if we could read this from one of the configuration files you created instead... (this is out of scope of this PR, just a thought).

Reply:
It could be possible but I'm not sure it would be good that conftest.py depends on the property file of another test framework(gradle)...